testnet/stacks-node/src/node.rs

```diff
@@ -181,6 +182,17 @@ fn spawn_peer(
 impl Node {
     /// Instantiate and initialize a new node, given a config
     pub fn new(config: Config, boot_block_exec: Box<dyn FnOnce(&mut ClarityTx) -> ()>) -> Self {
+        let use_test_genesis_data = if config.burnchain.mode == "mocknet" {
+            // When running in mocknet mode allow the small test genesis chainstate data to be enabled.
+            if env::var("BLOCKSTACK_USE_TEST_GENESIS_CHAINSTATE") == Ok("1".to_string()) {
```

Comment:
Instead of making this an envar, could we add this as a config option?

Reply:
We could, although env vars are easier to deal with in the tooling than config files. How strongly do you prefer config file vs env var?

Reply:
It would be nice for the competition to set this as a config option

Reply:
Are we okay with allowing both methods? Config or env var

Reply:
@jcnelson @friedger Please see latest commit where support for both config toml and env var is added.

Reply:
Great

Reply:
Both are fine.